### ISSUE TITLE
Max height/width shouldn't exist. Let us make it take up the whole screen if we want

### ISSUE THREAD

**Issue by cow1337killer3** [rpkamp/chrome-minimal-bookmarks-tree#156]
There's no reason why the max height should be 600px. That is absurd. 

And while I'm here, I might as well also mention:
1. When first installing, Bookmarks Bar folder should be expanded by default. Basic user experience stuff.  "Start with all folders closed" should also close everything except for the root Bookmarks Bar folder which should be expanded.
2. Dark/light theme should default to OS setting when first installed
3. "When clicking "Open all", open all bookmarks in all subfolders as well." --- Whatever this is set to, the opposite should also be an additional context menu option when right-clicking a folder. Or just get rid of the setting and have both as context menu options, doesn't really matter.
4. Let us change the font size, along with the vertical spacing between items, so we can make the view as compact/comfortable as we wish.
5. Give us a setting that will auto-expand a folder if we hover over it for X milliseconds (which we can also specify)

This one will be hard, but:
6. Should have another setting that allows us to hover over folders and have another menu pop up ontop. But double clicking a folder will "go" to that one and either reset the main window to expand the tree to there, or just make that one the "root" folder with a breadcrumb above to navigate back.

**Comment by rpkamp** [comment]
> There's no reason why the max height should be 600px. That is absurd.

I agree, but it's a limitation on Chrome's end, not on mine 🙁 I'll see if it's changed since I last looked, but I wouldn't get my hopes up...

I'll look into your other suggestions, but please note I do not have loads of time to work on this, so it might not happen soon, or indeed at all. 

I do really like the default dark/light OS settings and the font size suggestions. I'll certainly look into those.
